app/jobs/update_ticket_links_job.rb

```diff
@@ -31,14 +37,35 @@ def relink_tickets(before_sha, after_sha)
       ticket.paths.each do |feature_review_path|
         next unless feature_review_path.include?(before_sha)
 
-        link_feature_review_to_ticket(ticket.key, feature_review_path, before_sha, after_sha)
+        update_path_and_link_feature_review_to_ticket(ticket.key, feature_review_path, before_sha, after_sha)
       end
     end
   end
 
-  def link_feature_review_to_ticket(ticket_key, old_feature_review_path, before_sha, after_sha)
+  def check_and_link_new_branch(full_repo_name, branch_name, after_sha)
+    if check_branch_for_ticket_and_link?(full_repo_name, branch_name, after_sha)
+      post_not_found_status(full_repo_name, after_sha)
+    else
+      post_error_status(full_repo_name, after_sha)
+    end
+  end
+
+  def check_branch_for_ticket_and_link?(full_repo_name, branch_name, after_sha)
+    ticket_key = extract_ticket_key_from_branch_name branch_name
```

Comment:
Just to keep the style consistent
```suggestion
    ticket_key = extract_ticket_key_from_branch_name(branch_name)
```